src/modes/survey/KPI.svelte

```diff
@@ -0,0 +1,42 @@
+<script>
```

Comment:
I didn't expect to see survey related changes here.  Can you comment on their purpose?

Reply:
refactoring to the key card style such that I can also use it to show dates instead of just numbers